### ISSUE TITLE
TypeError: _document.createEventObject is not a function. (In '_document.createEventObject()', '_document.createEventObject' is undefined)

### ISSUE THREAD

**Issue by iamrommel** [expo/sentry-expo#43]
I intentionally throw error on one of my component, using React ErrorBoundary too to catch the generic error and display a better UI, but  everytime i do `Sentry.captureException` I always got the error below.

Maybe it is related to https://github.com/getsentry/sentry-javascript/issues/954 and setting `Raven._hasDocument = false;` maybe will fix it. If feel the error is coming from the document object which is not present on react-native. 

I am using
```
"sentry-expo": "^1.11.0"
 "expo": "^31.0.6",

```
```
[11:17:08] TypeError: _document.createEventObject is not a function. (In '_document.createEventObject()', '_document.createEventObject' is undefined)

This error is located at:
    in ErrorBoundary (at App.js:76)
    in RCTView (at View.js:44)
    in Container (at connectStyle.js:384)
    in Styled(Container) (at App.js:75)
    in Provider (at App.js:74)
    in ApolloProvider (at App.js:73)
    in RCTView (at View.js:44)
    in Container (at connectStyle.js:384)
    in Styled(Container) (at App.js:72)
    in RCTView (at View.js:44)
    in Root (at connectStyle.js:384)
    in Styled(Root) (at App.js:71)
    in StyleProvider (at App.js:70)
    in App (at registerRootComponent.js:17)
    in RootErrorBoundary (at registerRootComponent.js:16)
    in ExpoRootComponent (at renderApplication.js:34)
    in RCTView (at View.js:44)
    in RCTView (at View.js:44)
    in AppContainer (at renderApplication.js:33)
- node_modules\react-native-sentry\node_modules\raven-js\src\raven.js:940:12 in _triggerEvent
- node_modules\react-native-sentry\node_modules\raven-js\src\raven.js:1653:23 in _handleStackInfo
- node_modules\react-native-sentry\node_modules\raven-js\src\raven.js:526:28 in captureException
- node_modules\react-native-sentry\lib\RavenClient.js:85:27 in captureException
- node_modules\react-native-sentry\lib\Sentry.js:133:66 in captureException
* js\config\sentry.js:17:28 in logToSentry
* js\components\common\ErrorBoundary.js:16:16 in componentDidCatch
- node_modules\react-native\Libraries\Renderer\oss\ReactNativeRenderer-dev.js:12820:29 in callback
- node_modules\react-native\Libraries\Renderer\oss\ReactNativeRenderer-dev.js:7004:16 in callCallback
- ... 20 more stack frames from framework internals
```

**Comment by brentvatne** [comment]
can you post to https://github.com/getsentry/react-native-sentry/issues?

**Comment by brentvatne** [comment]
i published sentry-expo@1.11.1 which adds the following lines to sentry-expo/index.js

```js
import Raven from 'raven-js';
Raven._hasDocument = false;
```

tested it and it worked as expected

![screen shot 2019-01-08 at 11 05 01 am](https://user-images.githubusercontent.com/90494/50852774-5dd5d500-1335-11e9-9909-7d1dc1aadb02.png)

**Comment by iamrommel** [comment]
Thank you very much for the update
